_docs/install/health.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: guide
+title: StorageOS Docs - Health
+anchor: install
+module: install/health
+---
+
+# Cluster health
+
+Various tools are available for checking on the status of a cluster.
+
+### Cluster members
+
```

Comment:
The `storageos cluster health` command got revamped  today - would like to include that too as it has lots more useful info.